src/Tizen.NUI/src/internal/WidgetImpl.cs

```diff
@@ -310,7 +310,12 @@ private bool SwigDerivedClassHasMethod(string methodName, global::System.Type[]
 
         private void SwigDirectorOnCreate(string contentInfo, global::System.IntPtr window)
         {
-            OnCreate(contentInfo, new Window(window, true));
+            Window ret = Registry.GetManagedBaseHandleFromNativePtr(window) as Window;
+            if (ret == null)
```

Comment:
is it ok not to add new sub window in NUI window list?
does GetWindowList() get list from native side?

Reply:
GetWindowList() get list from native side, so i think it is fine.